ccd-definition/CaseEventToFields/CareSupervision/uploadAdditionalDocuments-nonprod.json

```diff
@@ -0,0 +1,144 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadAdditionalApplications",
+    "CaseFieldID": "additionalApplicationsLabel",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "N",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-additional-applications/get-fee/mid-event"
```

Comment:
do we need to get fee on first page? I though that fee may be affected by selection on 2nd page so having it calculated here is incorrect

Reply:
we agreed with gail my story would just contain the existing c2 fee's (with/without notice).. @aliveni's pr contains the logic which will then take into account the orders on the c2 page and the one after.